dspace/config/modules/elsevier-sciencedirect.cfg

```diff
@@ -0,0 +1,28 @@
+# Api key to be able to make the calls to retrieve the articles, this will need to be requested by the appropriate instance
+api.key = ${elsevier.api.key}
+
+# This represents the base url to use for the retrieval of an article
+api.article.url=http://api.elsevier.com/content/article
+# The base of rest endpoints to represent identifiers and entitlement status associated with requested full text articles
+api.entitlement.url=http://api.elsevier.com/content/article/entitlement
+# This represents retrieval of a full text article by PII (Publication Item Identifier).
+api.pii.url=//api.elsevier.com/content/article/pii/
+# The search interfaces associated with ScienceDirect
+api.scidir.url=http://api.elsevier.com/content/search/scidir
+# Url to base later rest calls on, such as retrieval based on PII etc
+ui.article.url=http://www.sciencedirect.com/science/article
+# Check statuses associated with the requested articles
+entitlement.check.enabled=false
+
+# Mapping between retrieved and saved metadata
+metadata.field.pii = elsevier.identifier.pii
+metadata.field.doi = dc.identifier
+
+# Whether or not to embed the display + its respective width and height
+embed.display=false
+embed.display.width=700
+embed.display.height=500
+
+# Define if the link to the embed display should be rendered above (top) or under (bottom) the file section on the item page.
+# Only supported by theme Mirage
+embed.link.position = top
```

Comment:
Because #1104 was just merged, all of these settings will need to be prefixed with "elsevier-sciencedirect." (e.g. `elsevier-sciencedirect.api.key`) since that is the name of the module configuration.